src/package.json

```diff
@@ -20,6 +20,8 @@
         "plugin.tscwatch": "npm run tsc -- -w",
         "demo.ios": "npm i && npm run tsc && cd ../demo && tns run ios --syncAllFiles",
         "demo.android": "npm i && npm run tsc && cd ../demo && tns run android --syncAllFiles",
+        "demo.reset": "cd ../demo && rm -rf platforms && cd ../src",
+        "plugin.prepare": "npm run tsc && cd ../demo && tns plugin remove nativescript-yourplugin && tns plugin add ../src && cd ../src",
```

Comment:
`npm run plugin.link` just wasn't working is why. But perhaps we have a hangout together sometime to look at the flows and see what is going to work well for all cases.